Jenkinsfile

```diff
@@ -105,9 +117,33 @@ timestamps {
   }
 }
 
-
-
-
+// Generic github API function
+// url : API url to call get command
+def authenticatedGithubCall(url) {
```

Comment:
Can we think of putting this method at upper/common layer? As this doesn't have any driver specific stuff.

Reply:
not sure where u mean for common layer? this is a similar function call as `wgetUpdateGithub`

Reply:
best to have a follow up task to investigate how to have a common library for all ecosystem repo functions and in that process document the pipeline workflow